t/class-wp-test-spy-rest-server.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+class WP_Test_Spy_REST_Server extends WP_REST_Server {
```

Comment:
We should credit the source of this class (I'm assuming it's from the WP-API test suite?)